less/index.less

```diff
@@ -54,6 +53,68 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
 }
 
+h1 {
+	font-size:4rem;
+	// margin-bottom:@butt-pillow;
+}
+
+// VARIABLES
+@butt-pillow:20px;
+
+// MIXINS
+.column-center() {
+	display:flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+// STYLES BEGIN
+.container {
+	border-left:1px solid black;
+	border-right:1px solid black;
+	margin:0 auto;
+	max-width:777px;
+	padding:3%;
+	.column-center();
+}
+
+header {
+	width:100%;
+	// border:1px solid yellow;
+	.column-center();
+	contact {
+		// border:1px solid red;
+		display:flex;
+		justify-content:center;
+		align-items: center;
+		p {
+			// border:1px solid cyan;
+			padding:1px;
+			margin:3px;
+			line-height: 2rem;
+			font-weight:bold;
+		}
+	}
+	nav {
+		// border:1px solid green;
+		display:flex;
+		width:50%;
+		justify-content: space-around;
+	}
+	a {
+		color:indigo;
+		&:hover {text-decoration:none;}
+	}
+}
```

Comment:
excellent use of nesting! now you can see first-hand why preprocessing is so awesome